x-pack/metricbeat/module/aws/sqs/_meta/fields.yml

```diff
@@ -5,35 +5,35 @@
   release: beta
   fields:
     - name: oldest_message_age.sec
-      type: long
+      type: scaled_float
       description: >
         The approximate age of the oldest non-deleted message in the queue.
-    - name: message.delayed
-      type: long
+    - name: messages.delayed
+      type: scaled_float
       description: >
         TThe number of messages in the queue that are delayed and not available for reading immediately.
-    - name: message.not_visible
-      type: long
+    - name: messages.not_visible
+      type: scaled_float
       description: >
         The number of messages that are in flight.
-    - name: message.visible
-      type: long
+    - name: messages.visible
+      type: scaled_float
       description: >
         The number of messages available for retrieval from the queue.
-    - name: message.deleted
-      type: long
+    - name: messages.deleted
+      type: scaled_float
```

Comment:
What I worry about making this scaled_float instead of long is that for large values this could become inaccurate. 

We had the discussion in the past but should we just round these to be long? Do you expect confusion if `_source` and the value to query for are not identical? I expect queries to be `> 100` and not really `= 100.3`

Reply:
Yeah you are right, I freaked out when I see messages.deleted value 0.3 didn't get collected because the `long` type rounded it. Will change it back. Thanks!